### PR TITLE
doc: update 2024-2025 roadmap

### PR DESCRIPTION
**Pull request by dbkinder**
Replace roadmap with updated content approved by OPEA TSC

### CODE REVIEW

**Comment by dbkinder** [comment]
@mkbhanda @chickenrae @hshen14 I can't assign reviewers for this PR so here are a few people I can mention in a comment :(

**Review by hshen14 — APPROVED** [review]

**Comment by dbkinder** [comment]
@hshen14 How do I get another reviewer and get this merged?

File: roadmap/2024-2025.md [hunk 1]
```diff
@@ -1,130 +1,252 @@
...
+  - Documentation
+  - Test automation script
+  - Telemetry
+
+- **Use Cases/Examples**
+  - Documentation
+  - Test automation script
+
+- **Cloud Native**
+  - Demo K8s resource management
```
Comment (mkbhanda) [inline roadmap/2024-2025.md:131]: this and the next one will be pushed out, longer term.

File: roadmap/2024-2025.md [hunk 2]
```diff
@@ -1,130 +1,252 @@
...
+- Vision: llava
+- Mixtral-8x22B
+
+### AI Tools Integration
+
+- VectorDB: Milvus
+
+### Deployment Type
+
+- Public Cloud AWS (Xeon CPU & NV GPU)
```
Comment (mkbhanda) [inline roadmap/2024-2025.md:149]: We already verified on AWS in July for CPUs. Just could not snag an Nvdia GPU (A100 or H100) on AWS.

File: roadmap/2024-2025.md [hunk 3]
```diff
@@ -1,130 +1,252 @@
...
+### Contribution
+
+- **Components**
+  - Fine-tuning E2E pipeline
+  - Knowledge Graph
+
+- **Use Cases/Examples**
+  - Fine-tuning (Lora)
+  - AI Agent (single Agent with text and Audio as user interface)
+  - Closed source LLM
```
Comment (mkbhanda) [inline roadmap/2024-2025.md:193]: How can we even offer this in an open source project?

File: roadmap/2024-2025.md [hunk 4]
```diff
@@ -1,130 +1,252 @@
...
+  - Static tuning on Resource management for deployment
+
+- **Evaluation & Others**
+  - CICD & Validation
+  - Eval: E2E (GenAIComps & GenAIExamples)
+
+### AI Models
+
+- LLM open: Grok 1
+- LLM Close: GPT3.5/4/4o, Claude 3/3.5
```
Comment (mkbhanda) [inline roadmap/2024-2025.md:206]: These could be connected into a RAG pipeline as external services via a URL and API Key

**Comment by dbkinder** [comment]
> @dbkinder some inline comments. If you say this is what we wrote in the sand when we started, I am fine with that, and will approve.

The words written were from the PPT slide, though I'm happy to incorporate your edits anyway.

**Review by chickenrae — APPROVED** [review]
Looks good to me, let's post

**Pull request merged**